Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -248,6 +248,8 @@ public void ProcRunner_ArgumentQuoting()
         }
 
 
+        // SONARMSBRU-136: TODO - re-enable the following test:
+        [Ignore]
```

Comment:
A number of tests were added as part of SONARMSBRU-127 that aren't applicable now.